docs/wiki/featurex/fortify.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: wiki
+title: fortify
+description: enables player to build fortifications.
+group: feature
+category: interaction
+parent: wiki
+mod: acex
+version:
+  major: 3
+  minor: 2
+  patch: 0
+---
+
+## 1. Overview
+
+### 1.1 Fortification
+
+Enable players to place down fortifications.
+
+### 1.2 Budget
+
+Enables missionmakers / admin to restrict amount of resources available.

```

Comment:
missionmakers -> mission makers
amount -> the amount

Reply:
Fixed. Thanks!